scene/3d/bone_attachment_3d.cpp

```diff
@@ -143,7 +143,6 @@ void BoneAttachment3D::_check_bind() {
 		if (bone_idx != -1) {
 			sk->connect(SceneStringName(skeleton_updated), callable_mp(this, &BoneAttachment3D::on_skeleton_update));
 			bound = true;
-			callable_mp(this, &BoneAttachment3D::on_skeleton_update);
```

Comment:
Then shouldn't we keep this line?

Reply:
See my comment here: https://github.com/godotengine/godot/pull/104209#issuecomment-2728543191
Creating a `callable_mp` object and not calling it (there is no .call() or .call_deferred() here) does nothing.

The previous version of this PR fixed it by making it always call `on_skeleton_update` outside of the `_check_bind`, but the update made 4 hours ago moved the `on_skeleton_update` improperly into the else statement, so now it is not called when `_check_bind` is called. Perhaps a misunderstanding occurred between the previous review comment and this one.

See here for how it was originally changed before the current version:
https://github.com/godotengine/godot/compare/97e83c18825e1f9aa6d8fa58205659257d8f5f83..5293ba9461d8d4b3296bcfe63cf1407c37570dda

Reply:
So should be like:
```
		if (bone_idx != -1) {
			sk->connect(SceneStringName(skeleton_updated), callable_mp(this, &BoneAttachment3D::on_skeleton_update));
			bound = true;
			on_skeleton_update();
```

or

```
	if (is_inside_tree()) {
		_check_bind();
	}
	on_skeleton_update();

	notify_property_list_changed();
```